Remotion/Web/Core/Utilities/IScriptUtility.cs

```diff
@@ -27,20 +27,10 @@ namespace Remotion.Web.Utilities
  public interface IScriptUtility
   {
     /// <summary>
-    /// Registers the include files required for the calls to <see cref="RegisterResizeOnElement"/>.
```

Comment:
We cannot remove all of the description. Maybe something like `Registers the client side scripts for the current theme.`